assets/sass/vendor/_mdc-dialog.scss

```
@@ -75,8 +66,10 @@
 	}
 
 	.mdc-dialog__dependencies {
+		font-size: $fs-label-sm;
+		letter-spacing: $ls-xs;
+		line-height: $lh-label-sm;
```

Now that I took a closer look at the Figma design with "Dev mode", it appears that this text actually uses `body/small` by default, but `label/small` for the bold "Note:" text. I think it is fine to disregard a different token for "Note:" as we don't have a specific selector for it, but while we're at it, let's also change this to `body/small`, sorry!

```suggestion
		font-size: $fs-body-sm;
		letter-spacing: $ls-xs;
		line-height: $lh-body-sm;
```